lambda/readPass/index.js

```diff
@@ -41,7 +41,7 @@ exports.handler = async (event, context) => {
       // Filter Date
       if (event.queryStringParameters.date) {
         const theDate = new Date(event.queryStringParameters.date);
-        var month = ('0' + theDate.getMonth() + 1).slice(-2);
+        var month = ('0' + (theDate.getMonth() + 1)).slice(-2);
```

Comment:
I think@danieltruong just came up with a nice one liner for formatting this way:
`const dateselector = theDate.toISOString().split('T')[0]`

(optional 🙂, also fine as is)

Reply:
This seems like a much cleaner solution. Going to implement this.

Also did a `findAll` and replaced other instances in the code where this could be cleaned up.